source/patterns/@aws-solutions-constructs/aws-fargate-stepfunctions/test/fargate-stepfunctions.test.ts

```diff
@@ -274,190 +287,27 @@ test('Check for custom log group props', () => {
     }
   });
 
-  checkForBaseResources(stack);
   expect(stack).toHaveResourceLike("AWS::Logs::LogGroup", {
     LogGroupName: logGroupName
   });
 });
 
+function createFargateConstructWithNewResources(stack: cdk.Stack, publicApi: boolean) {
+  return new FargateToStepfunctions(stack, 'test-construct', {
```

Comment:
This is the actual setup that's being tested - I don't think saving 10 lines of code in a test at the expense of hiding the the target of the test is a good tradeoff.

Let's leave it like this for now - but let's discuss the idea with the Clean Code group next week to see what a larger audience thinks going forward.